src/Microsoft.Sbom.Api/Config/ConfigSanitizer.cs

```diff
@@ -132,6 +147,17 @@ private void ValidateBuildDropPathConfiguration(IConfiguration configuration)
                 throw new ValidationArgException($"Please provide a value for the BuildDropPath (-b) parameter to generate the SBOM.");
             }
         }
+        else if (configuration.ManifestToolAction == ManifestToolActions.ValidateFormat)
+        {
+            if (configuration.SbomPath.Value != null)
+            {
+                return;
+            }
+            else
+            {
+                throw new ValidationArgException($"Please provide a value for the SbomPath (-sp) parameter to validate the SBOM.");
```

Comment:
nit: it seems like we have this check/ throw this error in 2 places, are they both necessary?

Reply:
Lol, nope, I pulled the logic out to a more sensibly named function and then didn't go back and delete it. Good catch!